dashboard/src/pages/apps/new.tsx

```diff
@@ -0,0 +1,564 @@
+  base: {
+    background: vars.bg.white2,
+    border: `1px solid ${vars.bg.white4}`,
+    borderRadius: '4px',
+    marginLeft: '4px',
+    padding: '8px 12px',
+
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '12px',
+  },
+})
+
+const RepositoriesContainer = styled('div', {
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '20px',
+  },
+})
+
+const SettingsContainer = styled('div', {
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '10px',
+  },
+})
+
+interface Website {
+  fqdn: string
+  path_prefix: string
+  strip_prefix: boolean
+  https: boolean
+  h2c: boolean
+  http_port: number
+  authenticationType: number
+}
+
+const EmptyWebsite: Website = {
+  fqdn: '',
+  path_prefix: '',
+  strip_prefix: false,
+  https: false,
+  h2c: false,
+  http_port: 0,
+  authenticationType: AuthenticationType.OFF,
+}
+
+interface WebsiteProps {
+  website: Website
+  setWebsite: (Website) => void
+  deleteWebsite: () => void
+}
+
+const Website = (props: WebsiteProps) => {
+  return (
+    <InputFormWebsite>
+      <InputForm>
+        <InputFormText>ドメイン名</InputFormText>
+        <InputBar placeholder='example.ns.trap.jp' />
+      </InputForm>
+      <InputForm>
+        <InputFormText>Path Prefix</InputFormText>
+        <InputBar placeholder='/' />
+      </InputForm>
+      <InputForm>
+        <InputFormCheckBox>
+          <Checkbox>Strip Path Prefix</Checkbox>
+          <Checkbox>https</Checkbox>
+          <Checkbox>(advanced) アプリ通信にh2cを用いる</Checkbox>
+        </InputFormCheckBox>
+      </InputForm>
+      <InputForm>
+        <InputFormText>アプリのHTTP Port番号</InputFormText>
+        <InputBar placeholder='80' />
+      </InputForm>
+      <InputForm>
+        <Radio
+          items={authenticationTypeItems}
+          selected={props.website.authenticationType}
+          setSelected={(auth) => props.setWebsite({ ...props.website, authenticationType: auth })}
+        />
+      </InputForm>
+      <InputFormWebsiteButton>
+        <Button onclick={props.deleteWebsite} color='black1' size='large'>
+          Delete website setting
+        </Button>
+      </InputFormWebsiteButton>
+    </InputFormWebsite>
+  )
+}
+
+const authenticationTypeItems: RadioItem[] = [
+  { value: 0, title: 'OFF' },
+  { value: 1, title: 'SOFT' },
+  { value: 2, title: 'HARD' },
+]
+
+interface PortPublication {
+  internet_port: number
+  application_port: number
+  protocol: number
+}
+
+const EmptyPortPublication: PortPublication = {
+  internet_port: 0,
+  application_port: 0,
+  protocol: PortPublicationProtocol.TCP,
+}
+
+interface PortPublicationProps {
+  portPublication: PortPublication
+  setPortPublication: (PortPublication) => void
+  deletePortPublication: () => void
+}
+
+const PortPublication = (props: PortPublicationProps) => {
+  return (
+    <InputFormWebsite>
+      <InputForm>
+        <InputFormText>Internet Port</InputFormText>
+        <InputBar placeholder='internet_port' />
```

Comment:
この辺のplaceholderが提供している情報が無（むしろミスリーディング）なので削除しちゃっていいと思います

ただ、これで言えばポート番号なので適当な番号を例示すると良さそう
```suggestion
        <InputBar placeholder='30000' />
```